src/content/en/updates/posts/2016/03/access-usb-devices-on-the-web.markdown

```diff
@@ -0,0 +1,295 @@
+---
+layout: updates/post
+title: "Access USB devices on the Web"
+description: "The WebUSB API has been created to provide an easy way to safely expose USB device services to the Web."
+featured_image: /web/updates/images/2016-03-02-access-usb-devices-on-the-web/web-usb-hero.jpg
+published_on: 2016-03-02
+updated_on: 2016-03-30
+authors:
+  - beaufortfrancois
+tags:
+  - news
+  - USB
+  - WebUSB
+  - IoT
+  - Arduino
+---
+
+If I tell you plain and simple "USB", there are good chances that you will
```

Comment:
This doesn't read right to me. Telling me something about USB would require you actually telling me something... I think you mean `If I said plain and simple "USB"`